### ISSUE TITLE
Copy review: Form detail landing page copydeck & new form fields

### ISSUE THREAD

**Issue by jenniferlee-dsva** [department-of-veterans-affairs/va.gov-team#9018]
## Copy review: Form detail landing page copydeck & new form fields

**See GitHub repo:** 
https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/find-va-forms-redesign/form-detail-landing-pages/WIP-content-docs

**Two documents:**
- [x] ` form-detail-lp-copydeck-consolidated-051220.docx`
This doc needs copy edit review. Please note: A lot of the copy will ultimately be pulled from the DB, so we need to be careful of where we edit. 

- [x] Related to field values: `Form-detail-page-content-fields-050820.docx`
I'd love your thoughts on the related to field values, especially the one that's 'not a VA form.'  @DanielleThierryUSDSVA 

@DanielleThierryUSDSVA  and @RLHecht  - let's touch base before you tackle this ticket. BC there's a DB and dynamic layer to these, I prefer to walk you through certain elements. 

Not urgent, but will need to get clean set copy in about 2 weeks. 

FYSA @stephorkandatx

**Comment by RLHecht** [comment]
@DanielleThierryUSDSVA @jenniferlee-dsva I've copyedited the copy deck. Nothing major in there--some commas, a few times the text didn't match the form name etc. [https://app.zenhub.com/files/133843125/f2f9c593-5faf-4201-a257-7cdf5f121fc8/download](https://app.zenhub.com/files/133843125/f2f9c593-5faf-4201-a257-7cdf5f121fc8/download)

**Comment by jenniferlee-dsva** [comment]
Thank you @RLHecht - I've updated the checkbox above. Will await feedback re other item. 

Also - I don't know if there's a separate LOE ticket (for entering the custom content fields for these top forms + the 450ish other basic forms). If there is, could you please be sure to provide LOE there; if not, could you provide that LOE here.

**Comment by stephorkandatx** [comment]
LOE/ Content Discovery ticket for entering the custom content at #9224

**Comment by DanielleThierryUSDSVA** [comment]
Hi @jenniferlee-dsva and @RLHecht. I've added my thoughts/suggestions to the form fields doc here:  https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-va-forms-redesign/form-detail-landing-pages/WIP-content-docs/Form-detail-page-content-fields-dt-suggestions-053020.docx

Also took a look at the consolidated copy deck and added some thoughts/suggestions there for consideration: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-va-forms-redesign/form-detail-landing-pages/WIP-content-docs/form-detail-lp-coydeck-consolidated-dt-suggestions-053020.docx

FYSA @stephorkandatx

**Comment by jenniferlee-dsva** [comment]
@DanielleThierryUSDSVA - thank you. I will take a look. Did you by any chance add your thoughts/suggestions to the version of the docuemnt that Randi commented on here: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/9018#issuecomment-632837240

**Comment by jenniferlee-dsva** [comment]
I've consolidated everyone's comments and updates in the copydeck - 6/2 updated version is here: 
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-va-forms-redesign/form-detail-landing-pages/WIP-content-docs/form-detail-lp-coydeck-consolidated-RH-DT-JL-060220.docx

@DanielleThierryUSDSVA - Thanks so much for your edits! I accepted 95% of them. There are a few places where we have to go with the prior version as-is, including the online tool CTA language. 

**RE tool - Per comment reply in doc:** 
This button will be dynamically generated so we won’t always have the opportunity to customize this text for each form. Couple of other constraints/requirements: 
-	Some forms go to a “file a claim” versus “apply” 
-	Some forms are not filing or applying at all, they are for managing benefits, such as updating your address or adding/removing dependents, so the apply/file language doesn’t work in those cases. 

Open to other ideas, but those are the constraints that this button name needs to meet – dynamic, not custom per form, some tools are apply/file and some tools are more granular like updating, adding, removing, requesting, and other task based actions.
